keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1188,46 +1188,40 @@ are included with the key provided. E.g.:
 
 #### Namespace scoped policy binding
```

Comment:
This section title is no longer accurate.

Can we move this "Namespace scoped policy binding" section down to potential future work and instead call this section something like "Per namespace policy params"?